site/en/blog/renderingng/index.md

```diff
@@ -0,0 +1,554 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'RenderingNG'
+description: >
+  In 2021, we will largely complete the process of designing, building and shipping RenderingNG, a next-generation rendering architecture for Chromium.
+subhead: >
+  Ready for the next generation of web content
+date: 2021-06-16
+authors:
+  - chrishtr
+alt: >
+  A pile of different devices.
+---
+
+I'm Chris Harrelson,
+the engineering lead for Rendering (HTML & CSS -> pixels) in Blink.
```

Comment:
```suggestion
the engineering lead for Rendering (transforming HTML and CSS to pixels) in Blink.
```

Reply:
@rachelandrew Please refer to our documentation styleguide for cases like this: https://developers.google.com/style/words-as-words?hl=en